config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
-  devise_for :user
+  mount_devise_token_auth_for 'User', at: 'auth'
+  resources :users
```

Comment:
We don't need any of these routes definitions! Lets keep only the one which mounts devise token auth